cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -439,6 +452,121 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
           resultTable()
         }
       }
+
+      section("The dbms `DATABASE MANAGEMENT` privileges", "administration-security-administration-dbms-privileges-database-management", "enterprise-edition") {
+        p("The dbms privileges for database management are assignable using Cypher administrative commands. They can be granted, denied and revoked like other privileges.")
+        p("include::dbms/database-management-syntax.asciidoc[]")
+
+        p("The ability to create databases can be granted via the `CREATE DATABASE` privilege. The following query shows an example of this:")
+        query("GRANT CREATE DATABASE ON DBMS TO databaseAdder", ResultAssertions((r) => {
+          assertStats(r, systemUpdates = 1)
+        })) {
+          statsOnlyResultTable()
+        }
+        p("The resulting role should have privileges that only allow creating databases:")
+        query("SHOW ROLE databaseAdder PRIVILEGES", assertPrivilegeShown(Seq(Map()))) {
```

Comment:
I like that we don't test the result for (almost) any of the privilege commands :P